src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1675,4 +1675,55 @@ private static int indexOfSameAllele(final VariantContext vc, final Allele allel
 
         return -1;
     }
+
+    /**
+     * Increment the number of called alternate and reference plus alternate alleles for a genotype
+     *
+     * @param calledAltAlleles number of called alternate alleles for all genotypes
+     * @param calledAlleles    number of called alleles for all genotypes
+     * @param genotype         genotype
+     * @return incremented called alleles
+     * @throws IllegalArgumentException if calledAltAlleles or genotype are null
+     */
+    public static int incrementChromosomeCountsInfo(final Map<Allele, Integer> calledAltAlleles, final int calledAlleles, final Genotype genotype) {
```

Comment:
Changed method to `int getChromosomeCounts(final Map<Allele, Integer> calledAltAlleles, final Genotype genotype)`. Clients add the return value to `calledAlleles`.